packages/rpc/lib/src/utils/exception/ln_client_error.dart

```diff
@@ -0,0 +1,7 @@
+class LNClientException implements Exception {
+  String message;
+  int code;
+  String data;
```

Comment:
```suggestion
  String? data;
```

This can be null, from spec
```
data
A Primitive or Structured value that contains additional information about the error.
This may be omitted.
The value of this member is defined by the Server (e.g. detailed error information, nested errors etc.).
```